plugins/guests/linux/cap/mount_virtualbox_shared_folder.rb

```diff
@@ -6,62 +6,85 @@ module VagrantPlugins
   module GuestLinux
     module Cap
       class MountVirtualBoxSharedFolder
+        @@logger = Log4r::Logger.new("vagrant::guest::linux::mount_virtualbox_shared_folder")
+
         extend Vagrant::Util::Retryable
 
         def self.mount_virtualbox_shared_folder(machine, name, guestpath, options)
           guest_path = Shellwords.escape(guestpath)
 
-          mount_commands = ["set -e"]
+          @@logger.debug "Mounting #{name} (#{options[:hostpath]} to #{guestpath})"
```

Comment:
Sorry for this nitpick since it may not jive with your style but I'm strongly against parenthesis-less function calls. I'm sure there are some littered throughout Vagrant mostly through contributions but stylistically I just find it easier to scan (physically) lines that have parens to know when they end. 

Are you comfortable with that?